docs/inventory.md

```diff
@@ -187,13 +187,43 @@ network_config:
       type: ethernet
       mac: "40:A6:B7:3D:B3:71"
       state: down
-  dns_resolver_ip: 10.40.0.100
+  dns_server_ips:
+    -  10.40.0.100
   routes:
     - destination: 0.0.0.0/0
       address: 172.17.0.1
       interface: bond0
 ```
 
+#### VLAN
```

Comment:
I think it might be better to break out the nokia support, post install patch and this doc update into separate PRs as they are unrelated.